### PR TITLE
[12.0][MIG] sales_team_operating_unit

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of sales_team_operating_unit module to version 12

### CODE REVIEW

File: sales_team_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "name": "Sales Team Operating Unit",
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "SerpentCS,"
+              "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/operating-unit",
+    "category": "Sales",
+    "depends": ["sales_team", "operating_unit"],
+    "data": [
```
Comment (nikul-serpentcs) [inline sales_team_operating_unit/__manifest__.py:16]: @bjeficent Sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

File: sales_team_operating_unit/views/crm_team_view.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            </field>
+        </field>
+    </record>
+
+    <record id="crm_team_view_form" model="ir.ui.view">
+        <field name="name">crm.team.form</field>
+        <field name="model">crm.team</field>
+        <field name="inherit_id" ref="sales_team.crm_team_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='company_id']" position="after">
```
Comment (nikul-serpentcs) [inline sales_team_operating_unit/views/crm_team_view.xml]: here direct use `field`

**Review by nikul-serpentcs — COMMENTED** [review]
however, commit history seems missing. Can you follow those steps to preserve the history? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

**Comment by bjeficent** [comment]
@nikul-serpentcs Sorry for the mistake, updates done. Could you review it please?

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

File: sales_team_operating_unit/models/crm_team.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        default=lambda self: self.env['res.users'].operating_unit_default_get()
+    )
+
+    @api.multi
+    @api.constrains('operating_unit_id', 'company_id')
+    def _check_company_operating_unit(self):
+        for team in self:
+            if (team.company_id and team.operating_unit_id and
+                    team.company_id != team.operating_unit_id.company_id):
+                raise UserError(_('Configuration error!\n\n'
```
Comment (AaronHForgeFlow) [inline sales_team_operating_unit/models/crm_team.py]: Can you remote the `!` symbol ? This error may hurt people's feelings :) Also, the \n is not needed.

**Review by AaronHForgeFlow — COMMENTED** [review]
Functional review is good. One one minor comment.

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review working fine! :+1:

**Pull request merged**